make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -218,6 +218,15 @@ ifeq ($(call isTargetOs, windows macosx), false)
       LIBAWT_XAWT_LIBS += -lpthread
     endif
 
+    # These are the object files provided by the awt native
+    # library. Do not include these external (non-awt_xawt library)
+    # objects in the static library. When statically linking the java
+    # launcher with all JDK and VM static libraries, we use the
+    # --whole-archive linker option. The duplicate objects in different
+    # static libraries cause linking errors due to duplicate symbols.
+    LIBAWT_XAWT_STATIC_EXCLUDE_OBJS := \
+      debug_assert.o debug_util.o debug_trace.o debug_mem.o systemScale.o
```

Comment:
Can this list be derived dynamically in some way? If they are all in the same directory, maybe we could base it on that instead of having to maintain an explicit list?

Reply:
With additional testing, I found 'debug_assert.o debug_util.o debug_trace.o debug_mem.o' are no longer included when building libawt_xawt,  using JDK head code base. We no longer need to be filtered out those for libawt_xawt.a. They were needed for JDK 11.